packages/legend-query-builder/src/stores/QueryBuilderResultState.ts

```diff
@@ -468,6 +475,7 @@ export class QueryBuilderResultState {
           this.queryBuilderState.applicationStore.telemetryService,
           reportWithState,
         );
+        this.setExecutionError(undefined);
```

Comment:
is this correct ? why are we setting undefined here if on line 460 we set the results. 
Shouldnt the reset of the execution error come at the start of when the "execute" button is clicked ?

Reply:
This is correct because if the query can be executed successfully then there is no error so we should set the ExecutionError to be undefined. 
 `reset of the execution error come at the start of when the "execute" button is clicked` should also work